docs/sources/mimir/get-started/about-grafana-mimir-architecture/index.md

```diff
@@ -11,16 +11,43 @@ weight: 5
 
 # Grafana Mimir architecture
 
-Grafana Mimir has a microservices-based architecture.
-The system has multiple horizontally scalable microservices that can run separately and in parallel.
-Grafana Mimir microservices are called components.
+Grafana Mimir has a microservices-based architecture. The system has multiple horizontally scalable microservices that can run separately and in parallel. Grafana Mimir microservices are called components.
 
 Grafana Mimir's design compiles the code for all components into a single binary.
-The `-target` parameter controls which component(s) that single binary will behave as.
 
-To get started easily, run Grafana Mimir in [monolithic mode](../../references/architecture/deployment-modes/#monolithic-mode) with all components running simultaneously in one process, or in [read-write mode](../../references/architecture/deployment-modes/#read-write-mode), which groups components into _read_, _write_, and _backend_ paths.
+The `-target` parameter controls which components that single binary behaves as.
 
-For more information, refer to [Deployment modes](../../references/architecture/deployment-modes/).
+To get started, you can run Grafana Mimir in [monolithic mode](../../references/architecture/deployment-modes/#monolithic-mode) with all components running simultaneously in one process. For more information, refer to [Deployment modes](../../references/architecture/deployment-modes/).
+
+## About ingest storage architecture
+
+Starting with Grafana Mimir 3.0, ingest storage architecture is stable and the default architecture for running Mimir.
+
+Ingest storage architecture is designed to enhance reliability, support growth, and enable use cases at an even larger scale. It enables you to run more robust and cost-effective Mimir deployments.
+
+{{< admonition type="note" >}}
+Classic architecture is still supported in Grafana Mimir version 3.0. However, this architecture is set to be deprecated in a future release. As a best practice, use ingest storage architecture when setting up a new deployment of Grafana Mimir.{{< /admonition >}}
+
+### How ingest storage architecture works
+
+Ingest storage architecture fundamentally changes how Grafana Mimir handles data. In classic architecture, ingester nodes are highly stateful. They combine in-memory data with local write-ahead logs (WALs) and participate in both writes and reads. This can lead to heavy queries disrupting live writes.
+
+Ingest storage architecture mitigates this issue through decoupling the read and write paths and significantly reducing the statefulness of ingesters. It instead uses object storage as a central data transfer pipeline between the write and read paths.
+
+Here's an overview of how ingest storage architecture works (WIP):
```

Comment:
@dimitarvdimitrov could you please help provide some quick bullet points about how the process works? I'm not sure we want to get into our use of Warpstream here, since that's just our specific implementation of it. What do you think?